app/models/product.rb

```diff
@@ -118,6 +118,10 @@ def self.group_by_type
     order(:type, :name).group_by { |product| product.class.model_name.human }
   end
 
+  def nonbillable?
```

Comment:
What do you think about adding a `skip_review?` method as well?  Just for consistency